### PR TITLE
Extend test case for `CasperServiceByJsonRPC`

### PR DESCRIPTION
**Pull request by gyroflaw**
Fix https://github.com/casper-ecosystem/casper-js-sdk/issues/175

TODO:
- [x] Native token transfer
- [x] ERC20 token deploy and query contract data

### CODE REVIEW

File: src/lib/Contracts.ts [hunk 1]
```diff
@@ -138,7 +137,7 @@ export class Contract {
     path: string[] = [],
     casperClient?: CasperClient,
     stateRootHash?: string
-  ): Promise<StoredValue> {
+  ): Promise<any> {
```
Comment (hoffmannjan) [inline src/lib/Contracts.ts:140]: this is pain, but for now we won't do anything better, right?

File: test/rpc/RPC.test.ts [hunk 2]
```diff
@@ -80,12 +96,9 @@ describe('RPC', () => {
     assert.equal(stateRootHash.length, 64);
   });
 
-  it('state_get_balance', async () => {
+  xit('state_get_balance', async () => {
```
Comment (hoffmannjan) [inline test/rpc/RPC.test.ts]: why its commented out?

File: test/rpc/RPC.test.ts [hunk 3]
```diff
@@ -139,14 +155,123 @@ describe('RPC', () => {
...
+      faucetKey.publicKey
+    );
     const [prefix, value, suffix] = uref.split('-');
     expect(uref).to.be.equal(faucetMainPurseUref);
     expect(prefix).to.be.equal('uref');
     expect(value.length).to.be.equal(64);
     expect(suffix.length).to.be.equal(3);
   });
 
+  xit('should transfer native token by session', async () => {
```
Comment (hoffmannjan) [inline test/rpc/RPC.test.ts]: and this too?

File: test/rpc/RPC.test.ts [hunk 4]
```diff
@@ -164,10 +289,8 @@ describe('RPC', () => {
   //   const eraSummary = await client.getEraInfoBySwitchBlockHeight(10);
```
Comment (hoffmannjan) [inline test/rpc/RPC.test.ts]: can we uncomment this code and make it working? I mean to check if we can find the deploys that was done before  in the erc20 section?

**Review by hoffmannjan — CHANGES_REQUESTED** [review]
Overall looking good, I would just expend a few test after erc20 token was deployed so we can check few other SDKs methods ☺️

File: src/lib/Contracts.ts (hunk 1 above)
Comment (gyroflaw) [reply src/lib/Contracts.ts:140]: yeah, you are right

**Review by hoffmannjan — APPROVED** [review]
LGTM :+1:

**Pull request merged**